apps/api/migrations/demo_data.sql

```diff
@@ -0,0 +1,29 @@
+DELETE FROM user_profiles as p WHERE p.user_id in (SELECT id FROM users WHERE telegram_id IN (1000, 2000, 3000));
```

Comment:
"demo_data.sql": I would stick to 0xx.sql naming convention to avoid issues with inconsistent order of execution when launching postgres via docker compose

Reply:
Renamed to 002.dql